src/Microsoft.Azure.Batch.SoftwareEntitlement.Client.Native/curl-7.55.1-1.patch

```diff
@@ -0,0 +1,24 @@
+diff --git a/ports/curl/CONTROL b/ports/curl/CONTROL
+index 1193e114..423b59de 100644
+--- a/ports/curl/CONTROL
++++ b/ports/curl/CONTROL
+@@ -1,4 +1,4 @@
+ Source: curl
+ Version: 7.55.1-1
+-Build-Depends: zlib, openssl, libssh2
++Build-Depends: openssl
+ Description: A library for transferring data with URLs
+diff --git a/ports/curl/portfile.cmake b/ports/curl/portfile.cmake
+index c770d3f2..032b2de0 100644
+--- a/ports/curl/portfile.cmake
++++ b/ports/curl/portfile.cmake
+@@ -44,7 +44,8 @@ vcpkg_configure_cmake(
+         -DENABLE_MANUAL=OFF
+         -DCURL_STATICLIB=${CURL_STATICLIB}
+         -DCMAKE_USE_OPENSSL=ON
+-    OPTIONS_DEBUG
++        -DCURL_ZLIB=OFF
++        -DCMAKE_USE_LIBSSH2=OFF    OPTIONS_DEBUG

```

Comment:
Are you missing a line feed before `OPTIONS_DEBUG`?

Reply:
If curl moved to `CMAKE_USE_OPENSSL=ON` by default, it may be nice to update the above patch so that we patch the file in a consistent manner (the end result is the same whether we patch the former or the latter).

Reply:
Fixed.